packages/palette/src/elements/Typography/Typography.tsx

```diff
@@ -106,6 +109,7 @@ export const Text = primitives.Text<TextProps>`
   ${space};
   ${textAlign};
   ${verticalAlign};
+  ${letterSpacing};
```

Comment:
If this is mixed in then a user would be able to do `<Sans letterSpacing="foo">` as a publically available prop, which we want to avoid.